ogre2/include/ignition/rendering/ogre2/Ogre2Text.hh

```diff
@@ -0,0 +1,104 @@
+/*
+ * Copyright (C) 2018 Open Source Robotics Foundation
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ */
+#ifndef IGNITION_RENDERING_OGRE2_OGRE2TEXT_HH_
+#define IGNITION_RENDERING_OGRE2_OGRE2TEXT_HH_
+
+#include "ignition/rendering/config.hh"
+#include "ignition/rendering/base/BaseText.hh"
+#include "ignition/rendering/ogre2/Ogre2Object.hh"
+#include "ignition/rendering/ogre2/Ogre2Geometry.hh"
+#include "ignition/rendering/ogre2/Export.hh"
+
+namespace ignition
+{
+  namespace rendering
+  {
+    inline namespace IGNITION_RENDERING_VERSION_NAMESPACE {
+    //
+    class Ogre2MovableText;
```

Comment:
This is the structure which is needed for text rendering with Ogre2. If we want to get text rendering working in the future it will be used, but currently it isn't used.

Reply:
`basically ignore anything` -> in this case, I think this can be ignored for now.

Reply:
Whoops commenting at the same time. Please ignore that.